docs/pages/build-reference/npm-cache-with-yarn.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: Using npm cache with Yarn v1
+hideTOC: true
+---
+
+By default, the EAS npm cache won't work with Yarn v1, because **yarn.lock** files contain URLs to registries for every package and Yarn does not provide any way to override it. The issue is fixed in Yarn v2, but the Yarn team does not plan to backport it to Yarn v1.
```

Comment:
```suggestion
By default, the EAS npm cache won't work with Yarn Classic, because **yarn.lock** files contain URLs to registries for every package and Yarn does not provide any way to override it. The issue is fixed in Yarn Berry (2+), but the Yarn team does not plan to backport it to Yarn Classic.
```

Naming tweaks.

Reply:
In the text, I would use Yarn 1 (Classic) and Yarn 2+ (Berry) to match Yarn's own phraseology.

Reply:
For consistency across docs, I'm using the pattern Kim suggested.